### ISSUE TITLE
Loss of function change is too slow

### ISSUE THREAD

**Issue by RoseLii** [balancap/SSD-Tensorflow#152]
INFO:tensorflow:global step 196220: loss = 2.2232 (0.407 sec/step)
INFO:tensorflow:global step 196230: loss = 3.2711 (0.382 sec/step)
INFO:tensorflow:global step 196240: loss = 3.2512 (0.405 sec/step)
INFO:tensorflow:global step 196250: loss = 3.7080 (0.392 sec/step)
INFO:tensorflow:global step 196260: loss = 3.2773 (0.409 sec/step)
INFO:tensorflow:global step 196270: loss = 3.0040 (0.391 sec/step)
INFO:tensorflow:global step 196280: loss = 3.1116 (0.413 sec/step)
train global step have 196280,but loss=3.  ,Why not convergence?please help me.

**Comment by shiyuangogogo** [comment]
@RoseLii Hi, your training loss is low enough for this code implementation.  Could you please tell me how did you train?
